Correct ITs for NonEmptyAtclauseDescription and JavadocTagContinuationIndentation

Vladlis opened the pull request:
In NonEmptyAtclauseDescriptionTest the only way I found possible to write "//warn" comments in the input file is
```java
/**
 	 * Some javadoc.  
 	 * @param c     	 */ //warn
```
But this would make the test incomplete, I guess.

romani commented:
Please ignore failure of codecov, we experimenting with it

mkordas commented on src/it/java/com/google/checkstyle/test/chapter7javadoc/rule713atclauses/JavadocTagContinuationIndentationTest.java:
```diff
@@ -2,58 +2,38 @@
-    public void noFinalizerExtendedTest() throws Exception {
-        
-        String msg = getCheckMessage(NoFinalizerCheck.class, "avoid.finalizer.method");
+    public void testDefault() throws Exception {
```
> `testDefault()`

I would make it more descriptive like `testWithDefaultConfiguration`

mkordas commented on src/it/java/com/google/checkstyle/test/chapter7javadoc/rule713atclauses/NonEmptyAtclauseDescriptionTest.java:
```diff
@@ -8,54 +10,45 @@
-            "45: " + msg,
-            "57: " + msg,
-            "69: " + msg,
+            //this is a case with description that is sequences of spaces
```
1) I think usually we put space after `//`
2) `sequences` -> `sequence`
3) We have multiple same comments here... My proposition is to create two input files, but one with spaces only (lines 27-39) and second with others (40-94). If they have proper test names, comments will become redundant.

mkordas commented on src/it/java/com/google/checkstyle/test/chapter7javadoc/rule713atclauses/NonEmptyAtclauseDescriptionTest.java:
```diff
@@ -8,54 +10,45 @@
+        String filePath = builder.getFilePath("InputNonEmptyAtclauseDescriptionCheck");
         
-        Integer[] warnList = builder.getLinesWithWarn(filePath);
+        Integer[] warnList = {27, 28, 29, 38, 39, 40, 76, 77, 78, 79, 80, 81, 90, 91, 92, 93, 94};
```
> `{27, 28, 29, 38, 39, 40, 76, 77, 78, 79, 80, 81, 90, 91, 92, 93, 94}`

Can we do something about it? Looks like duplication - any change in input file would force us to update line numbers in two places.

mkordas commented:
@Vladlis 

My review is done. Please fix the issues and then @romani can merge. Thanks for fixing that, these tests were ridiculous!

romani commented:
@Vladlis, please fix issues from mkordas and let me validate your code.

Vladlis commented:
> I would make it more descriptive like ```testWithDefaultConfiguration```

fixed
> My proposition is to create two input files

done
> Looks like duplication

fixed

romani commented:
merged as FF
